Formatted the Readme markdown

pablomayobre opened the pull request:
Fixed rendering issues in Github.

Please note that the links are still valid, cause Github automatically generates links for all headers.

PS: The rich diff is horrible!

tesselode commented on readme.md:
````diff
@@ -110,17 +108,16 @@ if Control.Vertical:pressed(1) then
 local tactile = require 'path.to.tactile'
 ```
 
 <a name="api"/>
````
i believe this line still needs to be removed?

pablomayobre committed:
Formatted the Readme markdown

Fixed rendering issues in Github.

Please note that the links are still valid, cause Github automatically generates links for all headers

pablomayobre commented:
Ups, I screwed up, fixed now!

Squashed because it made no sense to have two commits for this only

tesselode commented:
awesome, thank you! 👍
